src/controllers/shared/common.resume.submission.controller.ts

```diff
@@ -85,7 +85,7 @@ export const getResumePage = async (req: Request, res: Response, next: NextFunct
  * @param overseaEntityId
  */
 const setWebApplicationData = (session: Session, appData: ApplicationData, transactionId: string, overseaEntityId: string) => {
-
+  // TODO map the 3 new NOCs once added to API and SDK - (trust control and 2 x jurisdiction nocs)
```

Comment:
I think this TODO comment can be removed. SDK and API work both covered by the API story (other JIRA ticket).

Reply:
removed the Todo